io.github.TTimo.GtkRadiant.json

```diff
@@ -0,0 +1,220 @@
+{
+  "app-id": "io.github.TTimo.GtkRadiant",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "3.28",
+  "sdk": "org.gnome.Sdk",
+  "command": "gtkradiant",
+  "finish-args": [
+    "--share=network",
+    "--share=ipc",
+    "--socket=x11",
+    "--socket=pulseaudio",
+    "--device=dri",
+    "--filesystem=host"
+  ],
+  "cleanup": [
+    "/include",
+    "/lib/pkgconfig",
+    "/share/pkgconfig",
+    "/share/aclocal",
+    "/man",
+    "/share/man",
+    "/share/gtk-doc",
+    "*.la",
+    "*.a",
+    "/gtkradiant/libjpeg.so*",
+    "/gtkradiant/libpng*",
+    "/gtkradiant/installs/*/.svn",
+    "/gtkradiant/installs/*/.git"
+  ],
+  "modules": [
+    {
+        "name": "scons",
+        "cleanup": [ "*" ],
+        "no-autogen": true,
+        "sources": [
+            {
+                "type": "archive",
+                "url": "https://netix.dl.sourceforge.net/project/scons/scons/3.0.1/scons-3.0.1.tar.gz",
+                "sha256": "24475e38d39c19683bc88054524df018fe6949d70fbd4c69e298d39a0269f173"
+            },
+            {
+                "type": "file",
```

Comment:
Simply switch to `buildsystem: simple` and define commands in `build-commands`. It's been years since hacking around Makefile was needed.